fe/src/main/java/org/apache/doris/planner/MaterializedViewSelector.java

```diff
@@ -109,9 +109,22 @@ public BestIndexInfo selectBestMV(ScanNode scanNode) throws UserException {
         long start = System.currentTimeMillis();
         Preconditions.checkState(scanNode instanceof OlapScanNode);
         OlapScanNode olapScanNode = (OlapScanNode) scanNode;
+
+        ConnectContext connectContext = ConnectContext.get();
+        if (connectContext != null && connectContext.getSessionVariable().isUseV2Rollup()) {
```

Comment:
If user set 'use v2 rollup', the best rollup will not be selected?

Reply:
Yes, if 'use v2 rollup' is true, only base index will be selected